cin_validator/rules/cin2022_23/rule_8870Q.py

```diff
@@ -0,0 +1,69 @@
+'''
+Rule number: 8870Q
+Module: Section 47
+Rule details: Where present, the <InitialCPCtarget> (N00109) should not be a Saturday, Sunday
+
+Rule message: Please check: The Target Date for Initial Child Protection Conference should not be a weekend
+
+'''
+from typing import Mapping
+
+import pandas as pd
+
+from cin_validator.rule_engine import rule_definition, CINTable, RuleContext
+from cin_validator.rule_engine import IssueLocator
+from cin_validator.test_engine import run_rule
+
+# Get tables and columns of interest from the CINTable object defined in rule_engine/__api.py
+
+Section47 = CINTable.Section47
+InitialCPCtarget = Section47.InitialCPCtarget
+
+# define characteristics of rule
+@rule_definition(
+    code='8870Q',
+    module=CINTable.Section47,
+    message="Please check: The Target Date for Initial Child Protection Conference should not be a weekend",
+    affected_fields=[InitialCPCtarget],
+)
+
+def validate(
+    data_container: Mapping[CINTable, pd.DataFrame], rule_context: RuleContext
+):
+    df = data_container[Section47]
+    print(df)
+
+    # <InitialCPCtarget> (N00109) should not be a Saturday, Sunday
+    # Convert column to date format
+    df[InitialCPCtarget] = pd.to_datetime(df[InitialCPCtarget], format='%d-%m-%Y', errors = 'coerce')
+    print(df)
+    # .weekday() returns the integer value for each day (0-6) with weekends being 5 and 6
+    failing_indices = df[df[InitialCPCtarget].dt.weekday >= 5].index
+    print(failing_indices)
```

Comment:
That's interesting. 
I didn't know about this datetime method. It's nice to find out it exists.